core/src/main/scala/com/salesforce/op/filters/FeatureDistribution.scala

```diff
@@ -91,6 +95,24 @@ case class FeatureDistribution
    */
   def fillRate(): Double = if (count == 0L) 0.0 else (count - nulls) / count.toDouble
 
+  /**
+   * Test whether the given distribution is Uniform, for detecting useless text hashes
```

Comment:
is this documentation block related to `chiSqUnifTest`?

Reply:
if so, there is a redundant line between the docs block and the function

Reply:
fixed!